Lib/copyreg.py

```diff
@@ -53,24 +53,27 @@ def _reconstructor(cls, base, state):
 
 def _reduce_ex(self, proto):
     assert proto < 2
-    for base in self.__class__.__mro__:
+    cls = self.__class__
+    for base in cls.__mro__:
         if hasattr(base, '__flags__') and not base.__flags__ & _HEAPTYPE:
             break
     else:
         base = object # not really reachable
     if base is object:
         state = None
     else:
-        if base is self.__class__:
-            raise TypeError("can't pickle %s objects" % base.__name__)
+        if base is cls:
+            raise TypeError(f"cannot serialize '{cls.__name__}' object")
         state = base(self)
-    args = (self.__class__, base, state)
+    args = (cls, base, state)
     try:
         getstate = self.__getstate__
     except AttributeError:
         if getattr(self, "__slots__", None):
-            raise TypeError("a class that defines __slots__ without "
-                            "defining __getstate__ cannot be pickled") from None
+            raise TypeError(f"cannot serialize '{cls.__name__}' object: "
+                            f"a class that defines __slots__ without "
+                            f"defining __getstate__ cannot be pickled"
```

Comment:
There is a missing space here.

Reply:
Sorry, could you be more specific?

Reply:
Ah, I see, the ending space on this line. Thanks!